runtime/src/bank.rs

```diff
@@ -3185,6 +3185,11 @@ impl Bank {
     }
 
     pub fn freeze(&self) {
+        let builtins = builtins::get();
+        if !self.check_builtins_exist(&builtins) {
+            warn!("builtins missing when freezing the bank");
+        }
+
```

Comment:
I am ok with deleting them.  But if such condition happens again, then this check may help narrow down the problem.  A compromise may be to have some debug mode in which more checking is done, but otherwise it is skipped.

Reply:
Alright, i deleted them.